src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -11,20 +11,22 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public HelpCommand() {}
+    public HelpCommand() {
+    }
 
     @Override
     public CommandResult execute() {
         return new CommandResult(
                 AddCommand.MESSAGE_USAGE
-                + "\n" + DeleteCommand.MESSAGE_USAGE
-                + "\n" + ClearCommand.MESSAGE_USAGE
-                + "\n" + FindCommand.MESSAGE_USAGE
-                + "\n" + ListCommand.MESSAGE_USAGE
-                + "\n" + ViewCommand.MESSAGE_USAGE
-                + "\n" + ViewAllCommand.MESSAGE_USAGE
-                + "\n" + HelpCommand.MESSAGE_USAGE
-                + "\n" + ExitCommand.MESSAGE_USAGE
+                        + "\n" + DeleteCommand.MESSAGE_USAGE
+                        + "\n" + ClearCommand.MESSAGE_USAGE
+                        + "\n" + FindCommand.MESSAGE_USAGE
+                        + "\n" + SortCommand.MESSAGE_USAGE
```

Comment:
This is good, make sure to update your `help` command as well, so that internally it also reflects the new commands to the user. But because of the chunk changes, it might be hard to see what changes you've actually made.